db/raw/add-hotel-plugin.sql

```diff
@@ -0,0 +1,1576 @@
+    approved timestamp without time zone,
+    leader_id uuid
+);
+
+
+ALTER TABLE public."group" OWNER TO rams_db;
+
+--
+-- Name: hotel_requests; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE hotel_requests (
+    id uuid NOT NULL,
+    attendee_id uuid NOT NULL,
+    nights character varying DEFAULT ''::character varying NOT NULL,
+    wanted_roommates character varying DEFAULT ''::character varying NOT NULL,
+    unwanted_roommates character varying DEFAULT ''::character varying NOT NULL,
+    special_needs character varying DEFAULT ''::character varying NOT NULL,
+    approved boolean DEFAULT false NOT NULL
+);
+
+
+ALTER TABLE public.hotel_requests OWNER TO rams_db;
+
+--
+-- Name: job; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE job (
+    id uuid NOT NULL,
+    type integer DEFAULT 252034462 NOT NULL,
+    name character varying DEFAULT ''::character varying NOT NULL,
+    description character varying DEFAULT ''::character varying NOT NULL,
+    location integer NOT NULL,
+    start_time timestamp without time zone NOT NULL,
+    duration integer NOT NULL,
+    weight double precision DEFAULT 1::double precision NOT NULL,
+    slots integer NOT NULL,
+    restricted boolean DEFAULT false NOT NULL,
+    extra15 boolean DEFAULT false NOT NULL
+);
+
+
+ALTER TABLE public.job OWNER TO rams_db;
+
+--
+-- Name: m_points_for_cash; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE m_points_for_cash (
+    id uuid NOT NULL,
+    attendee_id uuid NOT NULL,
+    amount integer NOT NULL,
+    "when" timestamp without time zone NOT NULL
+);
+
+
+ALTER TABLE public.m_points_for_cash OWNER TO rams_db;
+
+--
+-- Name: merch_pickup; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE merch_pickup (
+    id uuid NOT NULL,
+    picked_up_by_id uuid NOT NULL,
+    picked_up_for_id uuid NOT NULL
+);
+
+
+ALTER TABLE public.merch_pickup OWNER TO rams_db;
+
+--
+-- Name: no_shirt; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE no_shirt (
+    id uuid NOT NULL,
+    attendee_id uuid NOT NULL
+);
+
+
+ALTER TABLE public.no_shirt OWNER TO rams_db;
+
+--
+-- Name: old_m_point_exchange; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE old_m_point_exchange (
+    id uuid NOT NULL,
+    attendee_id uuid NOT NULL,
+    amount integer NOT NULL,
+    "when" timestamp without time zone NOT NULL
+);
+
+
+ALTER TABLE public.old_m_point_exchange OWNER TO rams_db;
+
+--
+-- Name: password_reset; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE password_reset (
+    id uuid NOT NULL,
+    account_id uuid NOT NULL,
+    generated timestamp without time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+    hashed character varying DEFAULT ''::character varying NOT NULL
+);
+
+
+ALTER TABLE public.password_reset OWNER TO rams_db;
+
```

Comment:
It looks like you had the ``magprime`` plugin installed when you made these migrations, since ``prev_season_supporter`` isn't part of the core repo.  Or possibly that was just left over in your local ``m13`` database from when you had it installed at some point, and it didn't get removed by a ``sep reset_uber_db`` because your current sent of plugins didn't include it.  Either way it probably shouldn't be here.